README.md

```diff
@@ -235,3 +235,21 @@ Our deployment system is triggered by changes to the Helm chart.  Typically, thi
 * Breaking changes: update major level (e.g., `v1.0.1` to `v2.0.0`)
 
 In addition to updating version in the Helm chart, update the Release Notes in `release_notes.html`.  Put the latest changes first, following the established format.
+
+
+### Configuring email
+To test email functionality as a developer, add the following to your local secrets `env` file:
```

Comment:
Since `docker-compose.yml` explicitly uses `.docker-compose_local_secrets.env`, the documentation should too.